src/cryptenroll/cryptenroll.c

```diff
@@ -515,9 +550,13 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Too many arguments, refusing.");
 
-        r = parse_path_argument(argv[optind], false, &arg_node);
-        if (r < 0)
-                return r;
+        if (streq(argv[optind], "auto")) {
```

Comment:
Hmm, this of course means that if people have a file "auto" they cannot use the tool to enroll things in it...

we could add --auto or so as switch, to avoid the ambiguity here?